docs/general/fields.md

```diff
@@ -0,0 +1,23 @@
+[Back to Guides](../README.md)
+
+# Fields
+
+If for a reason, you need to override the fields defined on serializer, you should use `fields`
+
+For example, if you have a serializer like this
+
+```ruby
+class UserSerializer < ActiveModel::Serializer
+  attributes :access_token, :first_name, :last_name
+end
+```
+
+and in a specific controller, you can't return the `first_name`, `fields` will help you
+
+```ruby
+class AnonymousController < ApplicationController
+  def create
+    render json: User.create(activation_state: 'anonymous'), fields: [:access_token], status: 201
```

Comment:
Same as for the other comment. This snippet is only valid for the `json` and `attributes` adapters. Could you either add a small note or add the missing snippet for the `json_api` adapter?